packages/core/src/interpreter.ts

```diff
@@ -1320,7 +1320,7 @@ export function interpret<
   TContext = DefaultContext,
   TStateSchema extends StateSchema = any,
   TEvent extends EventObject = EventObject,
-  TTypestate extends Typestate<TContext> = any
+  TTypestate extends Typestate<TContext> = { value: any; context: TContext }
```

Comment:
Future: we should alias `{ value: any; context: TContext }` as `AnyTypestate<TContext>`.

Reply:
Yeah, was thinking about that but I have also noticed that instinctively I would declare such a type as:
```ts
type AnyTypestate<TContext> = { value: StateValue; context: TContext }
```
but from what I have checked when working on this PR this doesn't work (not sure why).